libraries/Microsoft.Bot.Builder/SimplePropertyAccessor.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder
+{
+    public class SimplePropertyAccessor<T> : IPropertyAccessor<T>
```

Comment:
IMNSHO, this should be `internal`. It's an implementation detail, nobody ever needs to know it exists.  It should also be `sealed`.